root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Framework/Js/common.js

```diff
@@ -734,6 +735,8 @@ function Fx_ShowModalScreen(url, style) {
         // 第2引数 = DialogFrame → DialogLoader.htmから起動するモーダル画面のURL
         // 第3引数 = 画面のスタイル(「項目1:値1;項目2:値2;…;項目n:値n」の形式) 
         var ret = window.showModalDialog(dialogFrameUrl.value, args, style);
+        ret = Fx_GetCookie("fx_window_returnValue");
```

Comment:
@daisukenishino 

At lines 737, you assign the return value of showModalDialog method to the variable ret.
Immediately after this, you assign the return value of Fx_GetCookie method to the variable ret again.

It is meaningless that you assign the return value of showModalDialog method to variable ret at lines 737.